application/locations-data-manager/chalicelib/service.py

```diff
@@ -8,23 +9,30 @@ def get_table_resource():
     return dynamodb_resource
 
 
+def get_table_name():
```

Comment:
Not for now maybe but could we move to this some utility function/class to avoid duplicating the code in each service